src/Akka.Persistence.SqlServer/sql-server.conf

```diff
@@ -30,6 +30,11 @@
 
 			sequential-access = on
 			
+			# By default, string parameter size in ADO.NET queries are set dynamically based on current parameter value size.
+			# If this parameter set to true, column sizes are loaded on journal startup from database schema, and 
+			# string parameters have constant size which equals to corresponding column size.
+			use-constant-parameter-size = false
```

Comment:
Decided to set this parameters `false` by default, so that nothing would change for the general user of this plugin (i.e. no db queries will start coming on journal setup). But if someone wants additional SQL Server query profiles optimization, he can enable new feature here.